src/webapi/Controllers/EvaluationController.cs

```diff
@@ -42,11 +42,61 @@ public Configuration GetAuthenticatedConfiguration()
         return authenticatedConfiguration;
     }
 
+    [HttpGet]
```

Comment:
This needs to be updated to [HttpGet("{performanceEvaluationId}")] so that it can retrieve the value from the GET request

Reply:
@stevenarnold-ed-fi Currently the verb template can receive the argument by parameter: For example: "https://localhost:7065/api/Evaluation?performanceEvaluationId=1"

You're asking to modify the template which effectively modifies the route for the parameter. For example:
"https://localhost:7065/api/Evaluation/1"

The current template can and does retrieves the value from the GET request. Please confirm this is what you want.

Reply:
Yes, this is what I want